FV3/coarse_graining/coarse_graining.F90

```diff
@@ -394,5 +452,64 @@ subroutine define_cubic_mosaic(domain, ni, nj, layout)
          istart2, iend2, jstart2, jend2, pe_start, pe_end, &
          symmetry=.true., name='coarse cubic mosaic')
   end subroutine define_cubic_mosaic
-  
+
+  subroutine compute_phalf_from_delp(delp, ptop, i_start, i_end, j_start, j_end, phalf)
+    integer, intent(in) :: i_start, i_end, j_start, j_end
```

Comment:
Is there a reason to input these args for this subroutine but just use the global `is`, `ie` etc in other subroutines?

Reply:
The reason is that I use this subroutine for both the fine and coarse grid, so I cannot hard-code their values.